WooCommerce/Classes/Extensions/UserDefaults+Woo.swift

```diff
@@ -65,6 +65,10 @@ extension UserDefaults {
 
         // Selected campaign objective saved for future campaigns
         case blazeSelectedCampaignObjective
+
+        // Whether the site is suspended on WordPress.com and can't be connected using Jetpack
+        //
+        case wpcomSiteSuspended
```

Comment:
Just a question for my learning, do we have any guidelines on where to use `UserDefaults` vs something like `GeneralAppSettings` in the app? in the custom fields project, I used `GeneralAppSettings`, and it needed a lot of boilerplate to my liking, so I'm not sure if I should've used `UserDefaults` instead 😅

Reply:
> do we have any guidelines on where to use UserDefaults vs something like GeneralAppSettings in the app?

Good question. I don't think there are any guidelines. 

Personally, I prefer user defaults when I have to update UI when the values change. Because, we can do [this](https://github.com/woocommerce/woocommerce-ios/pull/14257/files#diff-5e4b4001de415046921cf84e4ce2b6c16646ae1bb2067aa6d9ec1fbfa2a855e5R499-R504) to listen to the value using Combine and update UI.